crates/db_schema/src/aggregates/structs.rs

```diff
@@ -87,21 +90,32 @@ pub struct PostAggregates {
   pub upvotes: i64,
   pub downvotes: i64,
   pub published: DateTime<Utc>,
-  /// A newest comment time, limited to 2 days, to prevent necrobumping  
+  #[serde(skip)]
+  /// A newest comment time, limited to 2 days, to prevent necrobumping
   pub newest_comment_time_necro: DateTime<Utc>,
   /// The time of the newest comment in the post.
+  #[serde(skip)]
   pub newest_comment_time: DateTime<Utc>,
   /// If the post is featured on the community.
+  #[serde(skip)]
   pub featured_community: bool,
   /// If the post is featured on the site / to local.
+  #[serde(skip)]
   pub featured_local: bool,
+  #[serde(skip)]
   pub hot_rank: f64,
+  #[serde(skip)]
   pub hot_rank_active: f64,
+  #[serde(skip)]
   pub community_id: CommunityId,
+  #[serde(skip)]
   pub creator_id: PersonId,
+  #[serde(skip)]
   pub controversy_rank: f64,
+  #[serde(skip)]
   pub instance_id: InstanceId,
   /// A rank that amplifies smaller communities
+  #[serde(skip)]
```

Comment:
featured_local, featured_community are used by lemmy-ui. 

I grepped and it *seems* like the others are good / redundant with the postview.

Reply:
Both those you can get from `Post`

Reply:
Ahh okay, thx.